packages/core/src/external/aws/lambda-logic/modify-resources.ts

```diff
@@ -0,0 +1,599 @@
+    ...updRes,
+    dose,
+    route,
+  };
+}
+
+export type SlimCondition = Omit<Condition, "name" | "category" | "clinicalStatus" | "code"> & {
+  name?: string | undefined;
+  category?: string | undefined;
+  clinicalStatus?: string | undefined;
+  reference?: Record<string, string | Partial<SlimPractitioner>>;
+};
+
+function slimCondition(res: Condition): SlimCondition {
+  const updRes = cloneDeep(res);
+  delete updRes.code;
+
+  return {
+    ...updRes,
+    name: getUniqueDisplaysString(res.code),
+    category: getUniqueDisplaysString(res.category),
+    clinicalStatus: isUselessStatus(getUniqueDisplaysString(res.clinicalStatus))
+      ? undefined
+      : getUniqueDisplaysString(res.clinicalStatus),
+  };
+}
+
+export type SlimOrganization = Omit<Organization, "address" | "name"> & {
+  name?: string | undefined;
+  address?: string | undefined;
+  reference?: Record<string, string>;
+};
+
+function slimOrganization(res: Organization): SlimOrganization {
+  const updRes = cloneDeep(res);
+  return {
+    ...updRes,
+    address: getAddressString(res.address),
+  };
+}
+
+export type SlimLocation = Omit<Location, "address" | "type"> & {
+  address?: string | undefined;
+  type?: string | undefined;
+  reference?: Record<string, string>;
+};
+
+function slimLocation(res: Location): SlimLocation {
+  const updRes = cloneDeep(res);
+  return {
+    ...updRes,
+    address: getAddressString(res.address),
+    type: getUniqueDisplaysString(res.type),
+  };
+}
+
+function cleanUpNote(note: string): string {
+  return note
+    .trim()
+    .replace(new RegExp(REMOVE_FROM_NOTE.join("|"), "g"), "")
+    .replace(/<ID>.*?<\/ID>/g, "")
+    .replace(/<styleCode>.*?<\/styleCode>/g, "")
+    .replace(/\s*\n\s*/g, "\n")
+    .replace(/\*{2,}/g, "*")
+    .replace(/_{2,}/g, " ");
+}
+
+function getQuantityString(quantity: Quantity | undefined): string | undefined {
+  if (!quantity) return undefined;
+  return `${quantity.value}${quantity.unit ? ` ${quantity.unit}` : ""}`;
+}
+
+function getAddressString(address: Address | Address[] | undefined): string | undefined {
+  if (!address) return undefined;
+
+  return toArray(address)
+    .map(addr => `${addr.line}, ${addr.city}, ${addr.state}`)
+    .join("\n");
+}
+
+function getUniqueDisplaysString(
+  concept: CodeableConcept | CodeableConcept[] | undefined
+): string | undefined {
+  return getUniqueDisplays(concept)?.join(", ");
+}
+
+function getLongestDisplay(
+  concept: CodeableConcept | CodeableConcept[] | undefined
+): string | undefined {
+  if (!concept) return undefined;
+
+  const uniqueDescriptors = new Set<string>();
+  const concepts = toArray(concept);
+  concepts.forEach(concept => {
+    const text = concept.text;
+    if (text) uniqueDescriptors.add(text.trim().toLowerCase());
+
+    concept.coding?.forEach(coding => {
+      if (coding.display) uniqueDescriptors.add(coding.display.trim().toLowerCase());
+    });
+  });
+
+  if (uniqueDescriptors.size === 0) return undefined;
```

Comment:
Could call `getUniqueDisplays()` and use the result to get the longest, eh?

Reply:
nice, ya! simplified the func